gpcontrib/arenadata_toolkit/src/arenadata_toolkit_guc.c

```diff
@@ -0,0 +1,295 @@
+			return false;
+
+		return true;
+	}
+
+	GUC_check_errmsg("cannot change tracking status outside the tracking_register_relstorages function");
+	return false;
+}
+
+void
+tf_guc_define(void)
+{
+	DefineCustomIntVariable("arenadata_toolkit.tracking_bloom_size",
+				   "Size of bloom filter in bytes for each tracked database",
+							NULL,
+							&bloom_size,
+							DEFAULT_BLOOM_SIZE,
+							MIN_BLOOM_SIZE,
+							MAX_BLOOM_SIZE,
+							PGC_POSTMASTER,
+							0,
+							NULL,
+							NULL,
+							NULL
+		);
+
+	DefineCustomIntVariable("arenadata_toolkit.tracking_db_track_count",
+							"Count of tracked databases.",
+							NULL,
+							&db_track_count,
+							DEFAULT_DB_TRACK_COUNT,
+							MIN_DB_TRACK_COUNT,
+							MAX_DB_TRACK_COUNT,
+							PGC_POSTMASTER,
+							0,
+							NULL,
+							NULL,
+							NULL
+		);
+
+	DefineCustomBoolVariable("arenadata_toolkit.tracking_is_db_tracked",
+							 "Is current database tracked.",
+							 NULL,
+							 &is_tracked,
+							 DEFAULT_IS_TRACKED,
+							 PGC_SUSET,
+							 0,
+							 &check_tracked,
+							 NULL,
+							 NULL);
+
+	DefineCustomBoolVariable("arenadata_toolkit.tracking_snapshot_on_recovery",
+							 "Return full snapshot at startup/recovery.",
+							 NULL,
+							 &get_full_snapshot_on_recovery,
+							 DEFAULT_GET_FULL_SNAPSHOT_ON_RECOVERY,
+							 PGC_SUSET,
+							 0,
+							 &check_get_full_snapshot_on_recovery,
+							 NULL,
+							 NULL);
+
+	DefineCustomIntVariable("arenadata_toolkit.tracking_drops_count",
+							"Count of max monitored drop events.",
+							NULL,
+							&drops_count,
+							DEFAULT_DROPS_COUNT,
+							MIN_DROPS_COUNT,
+							MAX_DROPS_COUNT,
+							PGC_POSTMASTER,
+							0,
+							NULL,
+							NULL,
+							NULL);
+
+	DefineCustomStringVariable("arenadata_toolkit.tracking_schemas",
+							   "Tracked schema names.",
+							   NULL,
+							   &tracked_schemas,
+							   DEFAULT_TRACKED_SCHEMAS,
+							   PGC_SUSET,
+							   0,
+							   &check_schemas,
+							   NULL,
+							   NULL);
+
+	DefineCustomStringVariable("arenadata_toolkit.tracking_relstorages",
+							   "Tracked relation storage types.",
+							   NULL,
+							   &tracked_rel_storages,
+							   DEFAULT_TRACKED_REL_STORAGES,
+							   PGC_SUSET,
+							   0,
+							   &check_relstorages,
+							   NULL,
+							   NULL);
+
+	DefineCustomStringVariable("arenadata_toolkit.tracking_relkinds",
+							   "Tracked relation kinds.",
+							   NULL,
+							   &tracked_rel_kinds,
+							   DEFAULT_TRACKED_REL_KINDS,
+							   PGC_SUSET,
+							   0,
+							   &check_relkinds,
+							   NULL,
+							   NULL);
+
+
+	DefineCustomIntVariable("arenadata_toolkit.tracking_worker_naptime_sec",
+							"Toolkit background worker nap time",
+							NULL,
+							&tracking_worker_naptime_sec,
+							DEFAULT_NAPTIME,
+							1,
+							MAX_NAPTIME,
+							PGC_POSTMASTER,
```

Comment:
```suggestion
							PGC_SIGHUP,
```

Thus we can use `\! gpstop -u` in the test instead of `\! gpstop -raq -M fast` for this GUC, which is a bit faster.